### PR TITLE
src/db/filename.rs test_generate_filename functions supports windows

### PR DESCRIPTION
**Pull request by AliceLanniste**
I add some code to test_generate_filename supports windows, `cargo test` output is Ok

### CODE REVIEW

File: src/db/filename.rs [hunk 1]
```diff
@@ -130,10 +130,22 @@ mod tests {
     use super::*;
 
     #[test]
-    #[cfg(unix)]
     fn test_generate_filename() {
         let dirname = "test";
-        let mut tests = vec![
+        let mut tests = if cfg!(windows) {
+        let tests_windows =  vec![
```
Comment (Fullstop000) [inline src/db/filename.rs]: I think maybe you could remove the redundant assignment to `tests_windows` and `tests_unix` like below?
```rust
let mut tests = if cfg!(windows) {
    vec![/* something */]
} else {
    vec![/* something */]
}
```

**Comment by Fullstop000** [comment]
Thank you for testing out the support of `wickdb` on windows. And it will be also very helpful if you could add windows version for `test_parse_filename`.

File: src/db/filename.rs (hunk 1 above)
Comment (AliceLanniste) [reply src/db/filename.rs]: ok，I will fix it

**Comment by AliceLanniste** [comment]
@Fullstop000  HI，I deleted repeate code in `filename/test_generate_filename` and supported `test_parse_filename` function  in windows

**Comment by AliceLanniste** [comment]
@Fullstop000  I'm sorry,I forget using `cargo fmt`

**Pull request merged**